Pod/Classes/BONChain.m

```diff
@@ -389,14 +389,14 @@ - (BONTagComplexStyles)tagComplexStyles
 
 - (void)appendLink:(id<BONTextable>)link
 {
-    [self appendLink:link separator:nil];
+    [self appendLink:link separatorTextable:nil];
 }
 
-- (void)appendLink:(id<BONTextable>)link separator:(NSString *)separator
+- (void)appendLink:(id<BONTextable>)link separatorTextable:(id<BONTextable>)separator
 {
-    if (separator.length > 0) {
+    if (separator.text.string.length > 0) {
```

Comment:
There's an `isEmpty` method (or at least, the name is something like that), which you should use instead, because this `text`'s string could be empty, but its `nextText` might not be, and that method takes care of it.